ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -394,14 +407,22 @@ void RSkComponentImage::requestNetworkImageData(ImageSource source) {
 
     // TODO : Parse request headers and retrieve caching details
 
-    cacheExpiryTime_ = std::min(responseMaxAgeTime,static_cast<double>(DEFAULT_MAX_CACHE_EXPIRY_TIME));
+    cacheExpiryTime_ = std::min(SECTOMILLSECCONVERTER(responseMaxAgeTime),static_cast<double>(DEFAULT_MAX_CACHE_EXPIRY_TIME));
```

Comment:
can we optimize usage of this duplicated code and use CurlRequest::shouldCacheData() itself .